modules/gathering-data-audit-logs.adoc

```diff
@@ -6,9 +6,12 @@
 ifeval::["{context}" == "gathering-cluster-data"]
 :support:
 endif::[]
-ifeval::["{context}" == "audit-log-view"]
+ifeval::["{context}" == "audit-log-view"] 
```

Comment:
This file is not used currently. We might need it later if/when Must Gather is added to custom metrics autoscaler